### PR TITLE
ss-133: Fix several visual defects on landing page

### PR DESCRIPTION
**Pull request by Evcat6**

### CODE REVIEW

**Review by teslmik — APPROVED** [review]

File: frontend/src/bundles/common/components/input/input.tsx [hunk 1]
```diff
@@ -17,11 +17,12 @@ import styles from './styles.module.scss';
 type Properties<T extends FieldValues> = {
     control: Control<T, null>;
     errors: FieldErrors<T>;
-    label: string;
+    label?: string;
     placeholder?: string;
     name: FieldPath<T>;
     type?: InputType;
-    className?: string;
+    input_className?: string;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/input/input.tsx:24]:
```suggestion
    inputClassName?: string;
```

File: frontend/src/bundles/common/components/input/input.tsx [hunk 2]
```diff
@@ -17,11 +17,12 @@ import styles from './styles.module.scss';
...
+    label_className?: string;
```
Comment (nikita-remeslov) [inline frontend/src/bundles/common/components/input/input.tsx:25]:
```suggestion
    labelClassName?: string;
```

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/input/input.tsx (hunk 2 above)
Comment (Evcat6) [reply frontend/src/bundles/common/components/input/input.tsx]: Resolved in [21ff6e3](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/142/commits/21ff6e343c8c4dca0ca77bf01a3f766999e3e475)

File: frontend/src/bundles/common/components/input/input.tsx (hunk 1 above)
Comment (Evcat6) [reply frontend/src/bundles/common/components/input/input.tsx]: Resolved in [21ff6e3](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/142/commits/21ff6e343c8c4dca0ca77bf01a3f766999e3e475)

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**